docs/en/_sidebar.md

```diff
@@ -11,26 +11,36 @@
   - [Overview](/en/platform-engineers/overview.md)
   - [Application CRD](/en/application.md)
   - [Definition CRD](/en/platform-engineers/definition-and-templates.md)
-  - [Auto-generated Schema](/en/platform-engineers/openapi-v3-json-schema.md)
-  - [Defining Components](/en/platform-engineers/component.md)
-  - [Defining Traits](/en/platform-engineers/trait.md)  
-<!-- - [Defining Cloud Service](/en/platform-engineers/cloud-services.md) -->
 
-  - Using CUE
+  - Visualization
+  - [Generate Forms from Definitions](/en/platform-engineers/openapi-v3-json-schema.md)
+
+  - Defining Components
+  
+  - CUE
+  - [How-to](/en/cue/component.md)
   - [Learning CUE](/en/cue/basic.md)
-  - [Define Components in CUE](/en/cue/component.md)
-  - [Define Traits](/en/cue/trait.md)
-  - [Advanced Features](/en/cue/status.md)
 
-  - Using Helm
-  - [Define Components in Chart](/en/helm/component.md)
+  - Helm
+  - [How-to](/en/helm/component.md)
   - [Attach Traits](/en/helm/trait.md)
   - [Known Limitations](/en/helm/known-issues.md)
   
-  - Using Raw Kube
-  - [Define Components With Raw K8s](/en/kube/component.md)
+  - Raw Template
+  - [How-to](/en/kube/component.md)
   - [Attach Traits](/en/kube/trait.md)
 
+  - Defining Traits
+  - [How-to](/en/cue/trait.md)
+  - [Patch Traits](/en/cue/patch-trait.md)
+  - [Status Write Back](/en/cue/status.md)
+  - [Advanced Features](/en/cue/advanced.md)
+
+  - Hands-on Lab
+  - [Debug, Test and Dry-run](/en/platform-engineers/debug-test-cue.md)
+  - [Defining KEDA as Autoscaling Trait](/en/platform-engineers/keda.md)  
+<!-- - [Defining Cloud Service](/en/platform-engineers/cloud-services.md) -->
```

Comment:
we have finished the cloud service docs now, please unquote it

Reply:
I have read the doc and I don't think it's ready, especially this one:

> ** Note: We highly recommend that you should split the cloud resource provision and consuming in different applications.** ** Because the cloud resources can have standalone Lifecycle Management.** 

The recommended way (at least for RDS use case) should be defining the workload and cloud service in same YAML and let the platform to handle everything.

Reply:
And the data binding mechanism demonstrated in that doc is too restricted and barely usable (i.e. you have to create a new `ComponentDefinition` for different Secret key). The definition as a template should be very generic.

The old approach (a template for `valueFrom`) makes more sense (many many charts have similar approach) if we do not have a generic service binding capability.

xref: https://github.com/vmware-labs/service-bindings